Sources/SpeziOpenAI/ChatView.swift

```diff
@@ -23,7 +23,7 @@ public struct ChatView: View {
     public var body: some View {
         ZStack {
             VStack {
-                MessagesView($chat, bottomPadding: $messageInputHeight)
+                MessagesView($chat, hideMessagesWithRoles: MessageView.Defaults.hideMessagesWithRoles, bottomPadding: $messageInputHeight)
```

Comment:
If we use a default value in the initializers, we can remove the argument here:
```suggestion
                MessagesView($chat, bottomPadding: $messageInputHeight)
```